CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 - Changed iterators over `Patch` and `Changes` data into custom types instead of standard collection iterators. (#393)
 - Fixed typo in `SparceListIndexKeys` and `SparceListIndexValues` (#398)
+- Changes key-indexes interface allowing to use borrowed types for the search operations. (#392)
```

Comment:
Changed? It's not obvious for me what tense should we use in changelog.

Reply:
Sorry, I added the reviewers too soon, but now it should be ready. :sweat_smile: